graphics/scaler/normal.cpp

```diff
@@ -164,6 +165,56 @@ void Normal4x(const uint8 *srcPtr, uint32 srcPitch, uint8 *dstPtr, uint32 dstPit
 		dstPtr += dstPitch4;
 	}
 }
+
+/**
+ * Trivial nearest-neighbor 5x scaler.
+ */
+template<typename Pixel>
+void Normal5x(const uint8 *srcPtr, uint32 srcPitch, uint8 *dstPtr, uint32 dstPitch,
+							int width, int height) {
+	uint8 *r;
+	const uint32 dstPitch2 = dstPitch * 2;
+	const uint32 dstPitch3 = dstPitch * 3;
+	const uint32 dstPitch4 = dstPitch * 4;
+	const uint32 dstPitch5 = dstPitch * 5;
+	int b = sizeof(Pixel);
+
+	assert(IS_ALIGNED(dstPtr, 2));
+	while (height--) {
+		r = dstPtr;
+		for (int i = 0; i < width; ++i, r += b * 5) {
+			Pixel color = *(((const Pixel *)srcPtr) + i);
+
+			*(Pixel *)(r + b * 0) = color;
```

Comment:
As this is a heavy loop, we should consider moving the repeating multiplications and additions into a const var - I suppose this should make it faster even for non optimized debug builds then as it saves a lot of instructions.